common/changes/@uifabric/experiments/master_2018-06-19-03-56.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/experiments",
```

Comment:
> "@uifabric/experiments", [](start = 20, length = 25)

I think we should start publishing to internal instead of experiments since experiments are actually exposed externally.

Reply:
We create these components in packages/experiments (based off whatever process we follow for LeftNav) , so npm run changes automatically publishes them to uifabric/experiments.  @sikrishn , let me know if we need to follow a different process here.

Reply:
We create these components in packages/experiments (based off whatever process we follow for LeftNav) , so npm run changes automatically publishes them to uifabric/experiments.  @Sivaraman Krishnan , let me know if we need to follow a different process here.

---
In reply to: [196473151](https://github.com/OfficeDev/office-ui-fabric-react/pull/5252#discussion_r196473151) [](ancestors = 196473151)

Reply:
It's fine for now. We should move these later.

Reply:
fabric internal is in VSO, github is for public and experiments

[https://office.visualstudio.com/ISS/ISS%20Team/_git/fabric-internal](https://office.visualstudio.com/ISS/ISS%20Team/_git/fabric-internal)

I think we should follow diff process to get our components into internal

---
In reply to: [196536893](https://github.com/OfficeDev/office-ui-fabric-react/pull/5252#discussion_r196536893) [](ancestors = 196536893,196473151)